wasm/src/programs/data/plaintext.rs

```diff
@@ -19,13 +19,17 @@ use crate::{
     Ciphertext,
     Field,
     Scalar,
+    from_js_typed_array,
+    native::LiteralNative,
     plaintext_to_js_value,
+    to_bits_array_le,
     types::native::{FromBytes, IdentifierNative, PlaintextNative, ToBytes},
 };
-use std::ops::Deref;
+use once_cell::sync::OnceCell;
+use snarkvm_console::prelude::{FromBits, ToBits};
```

Comment:
We'll define "EcoSystem" imports as imports from `snarkvm` or `crate` (i.e. SDK imports) and "Foreign" imports as any other crate.

For ecosystem imports, `crate` imports should go first and `snarkvm` imports should go second after crate without a space. These should all go alphabetically.

Foreign imports but should be grouped together alphabetically as well and should begin after EcoSystem imports with a single space.

```suggestion
use snarkvm_console::prelude::{FromBits, ToBits};
```